### ISSUE TITLE
Issue when using INFRACOST_TERRAFORM_CLOUD_TOKEN and multiple projects in a config file

### ISSUE THREAD

**Issue by keslerm** [infracost/infracost#609]
I have a very simple project file:

```
version: 0.1

projects:
  - path: production
  - path: staging
```

And when using the env variable to set the TF CLOUD credentials, the first project will run but the second project fails.

The output is:

```
time="2021-04-23T11:31:19-04:00" level=info msg="Detected Terraform directory at production"
time="2021-04-23T11:31:20-04:00" level=warning msg="Can't sync usage file as it's not specified"
time="2021-04-23T11:31:20-04:00" level=debug msg="Creating temporary config file for Terraform credentials"
time="2021-04-23T11:31:20-04:00" level=debug msg="Writing Terraform credentials to temporary config file /tmp/945025377"
time="2021-04-23T11:31:20-04:00" level=info msg="starting: Running terraform plan"
time="2021-04-23T11:31:20-04:00" level=info msg="Running command: /home/rridley/.asdf/shims/terraform plan -input=false -lock=false -no-color -out=/tmp/tfplan326797644"
....
time="2021-04-23T11:32:04-04:00" level=info msg="Detected Terraform directory at staging"
time="2021-04-23T11:32:04-04:00" level=warning msg="Can't sync usage file as it's not specified"
time="2021-04-23T11:32:04-04:00" level=debug msg="Creating temporary config file for Terraform credentials"
time="2021-04-23T11:32:04-04:00" level=debug msg="TF_CLI_CONFIG_FILE is set, copying existing config from /tmp/945025377 to config to temporary config file /tmp/333311547"
```

I snipped the logs that didn't seem relevant

I see the file 333311547 get created, but ultimately it's empty when I check it after it fails.

**Comment by aliscott** [comment]
Thanks @keslerm. I was able to reproduce and have fixed in https://github.com/infracost/infracost/pull/610.

**Comment by keslerm** [comment]
@aliscott excellent! thank for the quick response, this is an fantastic tool!

**Comment by alikhajeh1** [comment]
@keslerm this has been fixed in [v0.8.5](https://github.com/infracost/infracost/releases/tag/v0.8.5)
